src/Pangraph.hs

```diff
@@ -133,6 +134,10 @@ vertexAttributes = vertexAttributes'
 edgeEndpoints :: Edge -> (Vertex, Vertex)
 edgeEndpoints = endpoints'
 
+-- | Like 'edgeEndpoints' but returns the 'VertexID's instead.
```

Comment:
This function is very easy to define, I am not convinced it should be in the API.